diff --git a/boltons/listutils.py b/boltons/listutils.py
index ffa9d94..06b9da2 100644
--- a/boltons/listutils.py
+++ b/boltons/listutils.py
@@ -308,7 +308,7 @@ class BarrelList(list):
                 li.sort()
             tmp_sorted = sorted(chain.from_iterable(self.lists))
             del self.lists[:]
-            self.lists.append(tmp_sorted)
+            self.lists[0] = tmp_sorted
             self._balance_list(0)
 
     def reverse(self):
