diff --git a/boltons/timeutils.py b/boltons/timeutils.py
index 82f201e..ebf1014 100644
--- a/boltons/timeutils.py
+++ b/boltons/timeutils.py
@@ -367,7 +367,7 @@ def daterange(start, stop, step=1, inclusive=False):
     m_step += y_step * 12
 
     if stop is None:
-        finished = lambda now, stop: False
+        finished = lambda t: False
     elif start <= stop:
         finished = operator.gt if inclusive else operator.ge
     else:
