diff --git a/boltons/timeutils.py b/boltons/timeutils.py
index 82f201e..bf7135f 100644
--- a/boltons/timeutils.py
+++ b/boltons/timeutils.py
@@ -368,7 +368,7 @@ def daterange(start, stop, step=1, inclusive=False):
 
     if stop is None:
         finished = lambda now, stop: False
-    elif start <= stop:
+    elif start < stop:
         finished = operator.gt if inclusive else operator.ge
     else:
         finished = operator.lt if inclusive else operator.le
