diff --git a/boltons/dictutils.py b/boltons/dictutils.py
index f913f29..b824e4d 100644
--- a/boltons/dictutils.py
+++ b/boltons/dictutils.py
@@ -338,12 +338,9 @@ class OrderedMultiDict(dict):
     def __eq__(self, other):
         if self is other:
             return True
-        try:
-            if len(other) != len(self):
-                return False
-        except TypeError:
+        elif len(other) != len(self):
             return False
-        if isinstance(other, OrderedMultiDict):
+        elif isinstance(other, OrderedMultiDict):
             selfi = self.iteritems(multi=True)
             otheri = other.iteritems(multi=True)
             zipped_items = zip_longest(selfi, otheri, fillvalue=(None, None))
