diff --git a/boltons/iterutils.py b/boltons/iterutils.py
index 433eaf5..38ec295 100644
--- a/boltons/iterutils.py
+++ b/boltons/iterutils.py
@@ -1080,7 +1080,7 @@ def default_exit(path, key, old_parent, new_parent, new_items):
     elif isinstance(new_parent, Set):
         vals = [v for i, v in new_items]
         try:
-            new_parent.update(vals)
+            new_parent.update(new_items)
         except AttributeError:
             ret = new_parent.__class__(vals)  # frozensets
     else:
